runner/src/internal.rs

```diff
@@ -34,6 +34,8 @@ pub struct Opt {
     commands: bool,
     #[structopt(long, help = "show logs of commands")]
     logs: bool,
+    #[structopt(long, help = "continue execution after error")]
```

Comment:
Interesting, what would be the use-case of this?

Reply:
From the commit description: 

"Allow control of whether `runner` should keep going even when it encounters
an error, analogous to the `set -e`/`set +e` flag for shell scripts.

Default to false so that an error immediately stops execution, for two reasons:

 - In interactive use (e.g. with `run-examples --client-variant=none`), any errors
   will scroll off-screen and then a stale version of the binaries (from a previous
   successful run) can be run without the user noticing.
 - In CI use, early termination leads to faster CI runs when there are failures."

Reply:
Thanks for quoting, I'm more interested in what the use-case would be for going on after issues have occurred? Defacto it makes them warnings, and it seems like it could be hard to decipher which of the subsequent errors are separate issues, and which are related to the previous error(s).

Reply:
Answered by @tiziano88's follow up https://github.com/project-oak/oak/pull/1475#discussion_r488503171